src/sass/_focus.scss

```diff
@@ -0,0 +1,86 @@
+@mixin applyFocusStyle() {
+  box-shadow: 0px 0px 0px 2px var(--focusColorInner),
+    0px 0px 0px 4px var(--focusColor), 0px 0px 0px 6px var(--focusColorOuter);
+  outline: none;
+  animation: focusShadow1 $durationQuick2 $easingEntry,
+    focusShadow2 $durationQuick2 $easingEntry calc($durationQuick2 / 2),
+    focusShadow3 $durationQuick2 $easingEntry $durationQuick2;
+
+  @media (forced-colors: active) {
+    outline-color: Highlight;
+    outline-width: 2px;
+    outline-offset: 6px;
+  }
+}
+
+@keyframes focusShadow1 {
+  0% {
+    box-shadow: 0px 0px 0px 0px var(--focusColorInner);
+  }
+
+  100% {
+    box-shadow: 0px 0px 0px 2px var(--focusColorInner);
+  }
+}
+
+@keyframes focusShadow2 {
+  0% {
+    box-shadow: 0px 0px 0px 1.8px var(--focusColorInner);
+  }
+
+  100% {
+    box-shadow: 0px 0px 0px 2px var(--focusColorInner),
+      0px 0px 0px 4px var(--focusColor);
+  }
+}
+
+@keyframes focusShadow3 {
+  0% {
+    box-shadow: 0px 0px 0px 2px var(--focusColorInner),
+      0px 0px 0px 3.8px var(--focusColor);
+  }
+
+  100% {
+    box-shadow: 0px 0px 0px 2px var(--focusColorInner),
+      0px 0px 0px 4px var(--focusColor), 0px 0px 0px 6px var(--focusColorOuter);
+  }
+}
+
+@mixin applyFocusTextStyle() {
+  outline-color: var(--focusColor);
+  outline-width: 2px;
+  outline-offset: 4px;
+  outline-style: auto;
+}
+
+@mixin applyFocus() {
+  transition: box-shadow $durationQuick1 $easingExit;
```

Comment:
why we need this transition + animation?

Reply:
@katarzynatobis not sure if you saw this comment

Reply:
Entry and exit animation differ significantly. The exit one is much simpler:
<img width="257" alt="Screenshot 2022-11-04 at 16 49 47" src="https://user-images.githubusercontent.com/60467496/200018827-b94cde61-c3b0-4c86-9016-fa3c9ea88ff4.png">
and this is why I used `transition` to animate. 
While the entry one is more difficult - each color has its own delay, easing and time, but it is still the same `box-shadow` property.